### PR TITLE
Potential Fields for SimPEG

### PR DESCRIPTION
**Pull request by rowanc1**
Incorporation of PF into the SimPEG branch.

- [x] Initial documentation
- [x] Analytics example
- [ ] Testing for integral problem (Grav and Mag)
- [ ] Example from notebook into simpeg examples
- [ ] Ensure existing functionality

```
from SimPEG import PF
```

When testing, please ensure that you do not have any `simpegPF` imports hanging around!

### CODE REVIEW

**Comment by rowanc1** [comment]
@sgkang The docs are still beautiful!

image

**Comment by fourndo** [comment]
@rowanc1 can you help me with the last Travis fails? I don't know how to fix it. Would be great to have this PR done soon so we can test the tutorial. Cheers.

**Comment by rowanc1** [comment]
I will try to get to this today!

**Comment by fourndo** [comment]
@rowanc1 Ok I am failing the coverage now, but will be writing more tests later. Can we force it in anyway?

File: tests/pf/assets/magnetics/Mesh_10m.msh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+47 47 20
```
Comment (lheagy) [inline tests/pf/assets/magnetics/Mesh_10m.msh]: we should move all of the assets to a database @rowanc1 : is setting up a bucket on gae a longer time-scale? if so, any suggestions for a short term soln?

File: tests/pf/test_sensitivity_PFproblem.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
+# import unittest
```
Comment (lheagy) [inline tests/pf/test_sensitivity_PFproblem.py]: why is this all commented out?

**Comment by rowanc1** [comment]
Pep8?!?!? Amazing.

**Comment by lheagy** [comment]
The files that were in tests/pf/assets are now in
- https://storage.googleapis.com/simpeg/tests/potential_fields/ActiveCells.dat
- https://storage.googleapis.com/simpeg/tests/potential_fields/Gaussian.topo
- https://storage.googleapis.com/simpeg/tests/potential_fields/Mesh_10m.msh
- https://storage.googleapis.com/simpeg/tests/potential_fields/ModelStart.sus 
- https://storage.googleapis.com/simpeg/tests/potential_fields/Obs_loc_TMI.obs
- https://storage.googleapis.com/simpeg/tests/potential_fields/PYMAG3D_inv.inp

**Comment by lheagy** [comment]
:) 
image

I have merged in the updates from dev. If all tests pass, then we are good to go!

**Comment by sgkang** [comment]
ha. that's pretty cool!


2016-08-15 11:10 GMT-07:00 Lindsey Heagy <notifications@github.com>:

**Comment by rowanc1** [comment]
Well done @fourndo @sgkang @lheagy

**Pull request merged**